syncer/syncer.go

```diff
@@ -223,9 +223,9 @@ type Syncer struct {
 	// `lower_case_table_names` setting of upstream db
 	SourceTableNamesFlavor utils.LowerCaseTableNamesFlavor
 
-	tsOffset                  atomic.Int64             // time offset between upstream and syncer, DM's timestamp - MySQL's timestamp
-	secondsBehindMaster       atomic.Int64             // current task delay second behind upstream
-	workerLagMap              map[string]*atomic.Int64 // worker's sync lag key:WorkerLagKey val: lag
+	tsOffset                  atomic.Int64    // time offset between upstream and syncer, DM's timestamp - MySQL's timestamp
+	secondsBehindMaster       atomic.Int64    // current task delay second behind upstream
+	workerLagArray            []*atomic.Int64 // worker's sync lag array, note that idx=0 is ddl lag and idx=1 is skip lag,sql worker lag idx=(queue id + 2)
```

Comment:
```suggestion
	workerLagArray            []*atomic.Int64 // worker's sync lag array, note that idx=0 is skip lag and idx=1 is ddl lag,sql worker lag idx=(queue id + 2)
```
According to the previous defined constants